task03:
  ros__parameters:
    service_name: '/trigger_service'  # Define the service name here
    default_string: 'No service available'
